modules/test-e2e/src/test/scala/org/knora/webapi/e2e/v2/SearchEndpointsGetSearchE2ESpec.scala

```diff
@@ -0,0 +1,133 @@
+/*
+ * Copyright © 2021 - 2025 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.knora.webapi.e2e.v2
+
+import sttp.client4.UriContext
+import zio.*
+import zio.test.*
+
+import org.knora.webapi.E2EZSpec
+import org.knora.webapi.e2e.v2.SearchEndpointsGetSearchE2ESpec.suite
+import org.knora.webapi.messages.store.triplestoremessages.RdfDataObject
+import org.knora.webapi.messages.util.rdf.JsonLDDocument
+import org.knora.webapi.messages.util.rdf.RdfModel
+import org.knora.webapi.sharedtestdata.SharedTestDataADM.*
+import org.knora.webapi.testservices.RequestsUpdates.addSimpleSchemaHeader
+import org.knora.webapi.testservices.ResponseOps.assert200
+import org.knora.webapi.testservices.ResponseOps.assert400
+import org.knora.webapi.testservices.TestApiClient
+import org.knora.webapi.util.TestDataFileUtil
+
+object SearchEndpointsGetSearchE2ESpec extends E2EZSpec {
+
+  override lazy val rdfDataObjects: List[RdfDataObject] = List(
+    RdfDataObject(path = "test_data/project_data/anything-data.ttl", name = "http://www.knora.org/data/0001/anything"),
+    RdfDataObject(
+      path = "test_data/project_data/incunabula-data.ttl",
+      name = "http://www.knora.org/data/0803/incunabula",
+    ),
+    RdfDataObject(path = "test_data/project_data/beol-data.ttl", name = "http://www.knora.org/data/0801/beol"),
+    RdfDataObject(
+      path = "test_data/project_ontologies/books-onto.ttl",
+      name = "http://www.knora.org/ontology/0001/books",
+    ),
+    RdfDataObject(path = "test_data/project_data/books-data.ttl", name = "http://www.knora.org/data/0001/books"),
+    RdfDataObject(
+      path = "test_data/generated_test_data/e2e.v2.SearchRouteV2R2RSpec/gravsearchtest1-admin.ttl",
+      name = "http://www.knora.org/data/admin",
+    ),
+    RdfDataObject(
+      path = "test_data/generated_test_data/e2e.v2.SearchRouteV2R2RSpec/gravsearchtest1-onto.ttl",
+      name = "http://www.knora.org/ontology/0666/gravsearchtest1",
+    ),
+    RdfDataObject(
+      path = "test_data/generated_test_data/e2e.v2.SearchRouteV2R2RSpec/gravsearchtest1-data.ttl",
+      name = "http://www.knora.org/data/0666/gravsearchtest1",
+    ),
+  )
+
+  private def loadFile(name: String) = TestDataFileUtil.readTestData("searchR2RV2", name)
```

Comment:
Serialized "correct" copies on disk – wonderful. One question though, is it possible to refresh the on-disk copy by flipping some boolean somewhere? Because for this schema of testing, this is the usual functionality.

Reply:
This functionality does not exist.

Reply:
Alright, probably also out of scope and perhaps also not touched by this PR.

I just recently run into this issue and I was slightly surprised that it's not there.